unix/client/ohostname.ml

```diff
@@ -1,7 +1,15 @@
 let () =
   let t = Udns_client_unix.create () in
+  let domain = Domain_name.of_string_exn Sys.argv.(1) in
   let res =
-    Udns_client_unix.gethostbyname t (Domain_name.of_string_exn Sys.argv.(1)) in
+    Udns_client_unix.gethostbyname t domain in
   match res with
   | Ok addr -> Fmt.pr "%a\n" Ipaddr.V4.pp addr
-  | Error (`Msg x) -> Fmt.epr "Failed to resolve: %s\n" x; exit 1
+  | Error _ ->
+    (* Here we fall back to trying to resolve the IPv6 address instead: *)
```

Comment:
looks good, but: should `ohostname` be similar to the unix `host` utility? if so, actually both A and AAAA queries should be performed (as well as a MX lookup)

Reply:
That was indeed the intention. Maybe it should also be renamed to `ohost` ? I always get those two mixed up...

Good point, that also seems useful, I'll do that.

Reply:
sounds good to call it `ohost`.